### ISSUE TITLE
Package name is `libcocos2dx` if building with Android Studio

### ISSUE THREAD

**Issue by OneSecure** [cocos2d/cocos2d-x#15587]
如下图所示，安卓手机上，安装包的显示名字是 “libcocos2dx” 而不是应用名称
[错误的包名字截图](http://ww3.sinaimg.cn/large/a8c38b53gw1f3ml2ojgs8j20dc0nqdhu.jpg)

其实修正相当简单：
- 找到 Android Studio 工程的  App 的 [AndroidManifest.xml](https://github.com/cocos2d/cocos2d-x/blob/v3/templates/cpp-template-default/proj.android-studio/app/AndroidManifest.xml) 文件，为 `<application>` 标签添加一个子属性  `android:label="@string/app_name"`
- 找到 cocos2d 的 [AndroidManifest.xml](https://github.com/cocos2d/cocos2d-x/blob/v3/cocos/platform/android/libcocos2dx/AndroidManifest.xml)  文件，删掉 `<application>` 标签里的子属性   `android:label="libcocos2dx"`  

就搞定了。

**Comment by minggo** [comment]
How did you create the Android project? Have you input your package name? Such as

``` sh
cocos new -p com.mycom.myapp -l cpp
```

**Comment by OneSecure** [comment]
Yes

**Comment by minggo** [comment]
Ok, i will have a try.

**Comment by minggo** [comment]
I did as you suggested, but the package name also showed `libcocos2dx`.

**Comment by minggo** [comment]
The package name shown correctly after restarting Android device. I will check if it breaks compatibility if using Android ADT.

**Comment by OneSecure** [comment]
简单粗暴地改成 'android:label="HelloCpp"' 怎样？我一开始就是这样改的。后来才改回上面的建议的。都能正确显示包名。不知你那里为啥不起作用。记得把第二条提及的属性删掉。我这里如果不删，会报错。

**Comment by minggo** [comment]
@OneSecure Please use English in future. If you are not confident with English, then you can translate it with Google translate, and paste Chinese and English together, such as

```
请问这个问题该怎么解决?
How to fix this issue?
```

**Comment by OneSecure** [comment]
OK.

**Comment by minggo** [comment]
Thanks. Because developers all over the world will join the discussion, and english is the only language that all developers know.

**Comment by minggo** [comment]
Fixed in #15598
